### PR TITLE
Move template info into its own file.

### PR DESCRIPTION
**Pull request by jholdstock**

### CODE REVIEW

File: template_fields.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package main
```
Comment (dajohi) [inline template_fields.go:5]: needs copyright comment from web.go.

File: template_fields.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	StakeVersionSuccess bool
+	// StakeVersionCurrent is the StakeVersion that has been seen in the recent block header.
+	StakeVersionCurrent uint32
+	// StakeVersionMostPopular is the most popular stake version that is NOT the current stake version.
+	StakeVersionMostPopular uint32
+	// StakeVersionMostPopularPercentage is the percentage of most popular stake versions out of possible votes.
+	StakeVersionMostPopularPercentage float64
+	// StakeVersionTimeRemaining is a string to show how much estimated time is remaining in the stake version interval.
+	StakeVersionTimeRemaining string
+	// Quorum and Rule Change Information
```
Comment (dajohi) [inline template_fields.go:69]: newline requested :)

**Review by dajohi — CHANGES_REQUESTED** [review]

File: template_fields.go (hunk 1 above)
Comment (jholdstock) [reply template_fields.go:5]: done

File: template_fields.go (hunk 2 above)
Comment (jholdstock) [reply template_fields.go:69]: done

**Review by dajohi — APPROVED** [review]
ok

**Pull request merged**